### PR TITLE
Freeplay Vocals Thing

### PR DESCRIPTION
**Pull request by megasssdev**
Its kinda useless,but cool

### CODE REVIEW

**Comment by Gui-iago** [comment]
auto loading on freeplay makes the game way more laggy than it should be.

**Comment by megakeh** [comment]
> auto loading on freeplay makes the game way more laggy than it should be.

wdym? it works perfectly for me.

**Comment by Gui-iago** [comment]
> > auto loading on freeplay makes the game way more laggy than it should be.
> 
> wdym? it works perfectly for me.

it lags with older hardware

**Comment by Gui-iago** [comment]
I'd probably make this optional or smth

File: source/FreeplayState.hx [hunk 1]
```diff
@@ -320,6 +320,7 @@ class FreeplayState extends MusicBeatState
 
 				FlxG.sound.list.add(vocals);
 				FlxG.sound.playMusic(Paths.inst(PlayState.SONG.song), 0.7);
+				FlxG.sound.list.add(vocals);
```
Comment (ActualMandM) [inline source/FreeplayState.hx:323]: What's the point if it's added literally right before the music plays?

**Comment by Stilic** [comment]
no

File: source/FreeplayState.hx (hunk 1 above)
Comment (megakeh) [reply source/FreeplayState.hx:323]: it doesnt actually play the vocals if its before the music thought

**Comment by fr459** [comment]
Wy you send soo much mensajes

File: source/FreeplayState.hx (hunk 1 above)
Comment (ActualMandM) [reply source/FreeplayState.hx:323]: vocals get played literally right after the playmusic command

https://github.com/ShadowMario/FNF-PsychEngine/blob/743898050850c2244cac2dae653681668e4f47a8/source/FreeplayState.hx#L322-L323

Comment (megakeh) [reply source/FreeplayState.hx:323]: well for me it does

**Comment by CeebeeOffical** [comment]
Yo bro make psych engine available on mobile 
`I also need devs for my fnf mobile `

**Comment by megasssdev** [comment]
> Yo bro make psych engine available on mobile `I also need devs for my fnf mobile `

what

**Comment by TheKitBoi** [comment]
> Yo bro make psych engine available on mobile `I also need devs for my fnf mobile `

https://github.com/jigsaw-4277821/FNF-PsychEngine-With-Android-Support

great job being off topic

**Pull request closed without merging**